### DISCUSSION TITLE
o1 API - Structured Outputs

### DISCUSSION THREAD

**Discussion by MrDedalus** [cline/cline#948] [Feature Requests]
o1 -Preview is now out of date, please update API model to o1. With the new ability for 01 API to work on structured outputs it could be an amazing addition to Cline. Cline is the best Ai coder, thank you.

**Comment by zyzhang1130** [comment]
Yeah I also tried to use OpenAI compatible to use o1 API, but o1 does not accept temperature parameter and hence will return error:
`400 Unsupported parameter: 'temperature' is not supported with this model.` How do I disable the temperature parameter? in OpenAI compatible?
